components/GoogleMapMarker.js

```diff
@@ -0,0 +1,270 @@
+import React, { PropTypes } from 'react';
+import {
+  View,
+  requireNativeComponent,
+  StyleSheet,
+  Platform,
+  NativeModules,
+  Animated,
+  findNodeHandle,
+} from 'react-native';
+
+const resolveAssetSource = require('react-native/Libraries/Image/resolveAssetSource');
+
+// eslint-disable-next-line react/prefer-es6-class
+const MapMarker = React.createClass({
```

Comment:
Yes exactly.  The other components are "siblings" in the file tree, but really "children" of `<MapView >`.